docs/source/conf.py

```diff
@@ -0,0 +1,82 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import recommonmark
+from recommonmark.transform import AutoStructify
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("./"))
+
+sys.path.insert(0, os.path.abspath("../"))
+
+# -- Project information -----------------------------------------------------
+
+project = "auto-dl"
+copyright = "2021, auto-dl"
+author = "auto-dl"
+
+# The full version, including alpha/beta/rc tags
+release = "v1_alpha"
```

Comment:
```suggestion
release = "v1-alpha"
```
If `-` is allowed then please use it else ignore the suggestion